src/io/multi_val_dense_bin.hpp

```diff
@@ -0,0 +1,168 @@
+/*!
+ * Copyright (c) 2020 Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE file in the project root for license information.
+ */
+#ifndef LIGHTGBM_IO_MULTI_VAL_DENSE_BIN_HPP_
+#define LIGHTGBM_IO_MULTI_VAL_DENSE_BIN_HPP_
+
+
+#include <LightGBM/bin.h>
+
+#include <cstdint>
+#include <cstring>
+#include <omp.h>
```

Comment:
@guolinke Shouldn't this include (here and in all other files) be handle via [`openmp_wrapper`](https://github.com/microsoft/LightGBM/blob/master/include/LightGBM/utils/openmp_wrapper.h) to not break [threadless version building](https://lightgbm.readthedocs.io/en/latest/Installation-Guide.html#build-threadless-version-not-recommended)?